action.yml

```diff
@@ -7,7 +7,7 @@ inputs:
   file-regex:
     description: The regex used to search for files where mermaid graph inlining will be applied
     required: true
-    default: "^index\.html?$"
+    default: "^index\\.html?$"
```

Comment:
The extension is fixed -- currently it makes sense to inline SVG only in html files

Reply:
why? I'd remove the hardcode and just use a regex, you never know what people's gonna match

Reply:
Uhm because if you inline the SVG in a file that doesn't support SVG rendering, you might have inconsistent behaviours... For instance, you cannot inline SVG in markdown files.
Therefore, I thought that it could make sense to force files to be HTML.